process_labels/loading_funcs/tanzania.py

```diff
@@ -100,6 +102,37 @@ def load_tanzania():
     return df
 
 
+def load_tanzania_ceo():
+
+    ceo_files = (DATASET_PATH / "tanzania" / "ceo_labels").glob("*.csv")
+
+    gdfs: List[geopandas.GeoDataFrame] = []
+    for filepath in ceo_files:
+        single_df = pd.read_csv(filepath)
+        single_df[RequiredColumns.GEOMETRY] = single_df["sample_geom"].apply(wkt.loads)
+        single_df["is_crop_mean"] = single_df.apply(
+            lambda x: x["Crop/non-Crop"] == "Cropland", axis=1
+        )
+        gdfs.append(geopandas.GeoDataFrame(single_df, crs="epsg:4326"))
+
+    df = pd.concat(gdfs)
+    df = df.groupby("plotid").agg(
+        {
+            RequiredColumns.LON: "first",
+            RequiredColumns.LAT: "first",
+            RequiredColumns.GEOMETRY: "first",
+            "is_crop_mean": "mean",
+        }
+    )
+
+    df[RequiredColumns.COLLECTION_DATE] = datetime(2022, 8, 8)
```

Comment:
The filename says 2019, but the collection date says 2022; which is the correct value?